charts/hono/Chart.yaml

```diff
@@ -15,9 +15,9 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 2.6.1
+version: 2.6.2
 # Version of Hono being deployed by the chart
-appVersion: 2.5.1
+appVersion: 2.6.0
```

Comment:
Changes corresponding to the renaming of the Command Router image (from `hono-service-command-router` to `hono-service-command-router-infinispan` are required as well, also concerning the native image).

Also the currently wrong `kafka.autoCreateTopicsEnable` and `kafka.deleteTopicEnable` properties in `values.yaml` should be replaced by adding 
```
      auto.create.topics.enable=true
      delete.topic.enable=true
```
to `kafka.controller.extraConfig`.

As a further point, the `profileOpenshift-values.yaml` could be corrected in terms of putting
```
  podSecurityContext:
    enabled: false
  containerSecurityContext:
    enabled: false
```
under `kafka.controller`.

Reply:
> Changes corresponding to the renaming of the Command Router image (from hono-service-command-router to hono-service-command-router-infinispan are required as well, also concerning the native image).

The images have also been tagged with the original names. As long as there is no alternative (Redis based) implementation, there is little need for changing the names (yet).

Reply:
Just for my understanding: the other issues are not related to (that is: caused by) the update to Hono 2.6.0, or am I mistaken?

Reply:
> The images have also been tagged with the original names. As long as there is no alternative (Redis based) implementation, there is little need for changing the names (yet).

Ok, I wasn't aware of that. I had tested the chart with locally built `2.6.0-SNAPSHOT` images (using the `honoImagesTag` helm install parameter) right before the 2.6.0 release and saw the need to adapt the Command Router image name with that (by setting `commandRouterService.imageName`). Maybe a corresponding hint can be added in the chart release notes.

> Just for my understanding: the other issues are not related to (that is: caused by) the update to Hono 2.6.0, or am I mistaken?

Yes, these are independent.